### PR TITLE
Mod publishing on release creation

### PR DESCRIPTION
**Pull request by sschr15**
Hopefully it's set up correctly. Please tell me what's wrong if it's not right

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -270,3 +274,41 @@ gitHooks {
...
+            uploadTask.dependsOn(tasks.remapJar)
+            uploadTask.dependsOn(project(":core").tasks.remapJar)
+        }
+    })
+
+    curseGradleOptions.forgeGradleIntegration = false
+}
+
+modrinth {
+    projectId.set("qkl")
```
Comment (Oliver-makes-code) [inline build.gradle.kts:304]: This should be the project id, lwVhp9o5

File: build.gradle.kts [hunk 2]
```diff
@@ -270,3 +274,41 @@ gitHooks {
...
+    versionName.set("[${libs.versions.minecraft.get()}] QKL $rootVersion + FLK $flkVersion + Kotlin ${project.libs.versions.kotlin.orNull}")
+    versionType.set("release")
+    dependencies { 
+        embedded.project("flk")
+    }
```
Comment (Oliver-makes-code) [inline build.gradle.kts:309]: Should contain qsl

File: build.gradle.kts [hunk 3]
```diff
@@ -270,3 +274,41 @@ gitHooks {
...
+        releaseType = "release"
+        addGameVersion(libs.versions.minecraft)
+        addGameVersion("Quilt")
+
+        mainArtifact(tasks.remapJar)
+        addArtifact(project(":core").tasks.remapJar)
+
+        relations(closureOf<CurseRelation> { 
+            embeddedLibrary("308769")
+        })
```
Comment (Oliver-makes-code) [inline build.gradle.kts:292]: Should also contain qsl

File: build.gradle.kts [hunk 4]
```diff
@@ -270,3 +274,41 @@ gitHooks {
...
+
+modrinth {
+    projectId.set("qkl")
+    versionName.set("[${libs.versions.minecraft.get()}] QKL $rootVersion + FLK $flkVersion + Kotlin ${project.libs.versions.kotlin.orNull}")
+    versionType.set("release")
+    dependencies { 
+        embedded.project("flk")
+    }
+
+    file.set(tasks.remapJar.get().archiveFile)
```
Comment (Oliver-makes-code) [inline build.gradle.kts:311]: Why does cf have core added but nor mr?

**Review by Oliver-makes-code — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 2 above)
Comment (PotatoPresident) [reply build.gradle.kts]: slug is valid

**Review by Oliver-makes-code — APPROVED** [review]

**Review by NoComment1105 — APPROVED** [review]
Looks solid. Good work

**Pull request merged**